test_data/makedata_suites/060_computed_values.js

```diff
@@ -0,0 +1,776 @@
+            ],
+            "fields" : {
+              "cv_field" : {
+              }
+            },
+            "includeAllFields" : true,
+            "storeValues" : "none",
+            "trackListPositions" : false
+          }
+        }
+      }
+
+      // this method will compare two outputs
+      checkComValProperties(`${view[0]}`, creationOutput, expected_output);
+
+
+      // creating testviewV2 allias
+      db._createView(view[1], "search-alias", {
+        "indexes": [
+          {
+            'collection': c[0],
+            'index': 'inverted'
+          },
+          {
+            'collection': c[1],
+            'index': 'inverted'
+          },
+          {
+            'collection': c[2],
+            'index': 'inverted'
+          },
+          {
+            'collection': c[3],
+            'index': 'inverted'
+          },
+          {
+            'collection': c[4],
+            'index': 'inverted'
+          },
+          {
+            'collection': c[5],
+            'index': 'inverted'
+          },
+          {
+            'collection': c[6],
+            'index': 'inverted'
+          },
+          {
+            'collection': c[7],
+            'index': 'inverted'
+          },
+          {
+            'collection': c[8],
+            'index': 'inverted'
+          },
+          {
+            'collection': c[9],
+            'index': 'inverted'
+          },
+          {
+            'collection': c[10],
+            'index': 'inverted'
+          },
+          {
+            'collection': c[11],
+            'index': 'inverted'
+          }
+        ]
+      })
+        
+
+      //-------------------------------------------------------x-------------------------------------------------------------
+      //inserting data to all collection
+      let data_array = [a1, a2, a3, a4, a5, a6, a7, a8, a9, a10, a11, a12];
+      let docsAsStr = fs.read(`${PWD}/makedata_suites/060_computed_value.json`);
+
+      // this function will read and insert and check all the neccessary data for the respective collection
+      data_array.forEach(col => {
+        col.save(JSON.parse(docsAsStr), { silent: true });
+
+        //this cmd will find one docs from the collection
+        let expected_field = col.all().limit(5).toArray();
+        //checking computed value field exit on the collection's doc
+        if (col === a1 || col === a2 || col === a7 || col === a8 || col === a9 || col === a11 || col === a12) {
+          if (expected_field[0].cv_field !== null) {   
+          } else {
+            throw new Error(`Computed value field missing from collection`);
+          }
+        } 
+        else if (col === a3) {
+          if (expected_field[0].cv_field_insert !== null) {
+          } else {
+            throw new Error(`Computed value field missing from collection`);
+          }
+        }
+        else if (col === a4 || col === a5) {
+          if (expected_field[2].cv_field !== null) {
```

Comment:
```suggestion
          if (expected_field[2].cv_field == null) {
```

Just compare it with null. And if it is null - then you go under this condition and throw a error.

Please fix the same issue in code below and above

Reply:
Done.